apify-api/openapi/components/schemas/actor-pricing-info/CommonActorPricingInfo.yaml

```diff
@@ -0,0 +1,29 @@
+title: CommonActorPricingInfo
+type: object
+required:
+    - apifyMarginPercentage
+    - createdAt
+    - startedAt
+properties:
+  apifyMarginPercentage:
+    type: number
+    description: "In [0, 1], fraction of pricePerUnitUsd that goes to Apify"
+  createdAt:
+    type: string
+    format: date-time
+    description: "When this pricing info record has been created"
+  startedAt:
+    type: string
+    format: date-time
+    description: "Since when is this pricing info record effective for a given Actor"
+  notifiedAboutFutureChangeAt:
+    type: string
+    format: date-time
+    nullable: true
+  notifiedAboutChangeAt:
+    type: string
+    format: date-time
+    nullable: true
+  reasonForChange:
+    type: string
+    nullable: true
```

Comment:
I don't think these fields are relevant publicly, I wouldn't have them here.

Reply:
I understand, but they are exposed, so they should be documented. I'll add `x-internal: true` so that it can be hidden in the docs.